marketplace/netlify-build/README.md

```diff
@@ -0,0 +1,25 @@
+# Netlify Build and Preview
+
+This sidebar extension allows you to build and preview up to 3 Netlify sites.
+
+This is an alternative approach to using a webhook to trigger a build. The advantage
+is that the editor gets control on when a build is executed instead of responding
+to automatic events such as `Entry.publish`.
+
+![Netlify Sidebar UI Sample](./assets/netlify-triggering.gif)
+
+## Requirements
++ A Netlify site
++ Netlify webhooks setup, [see their documentation](https://www.netlify.com/docs/webhooks/)
+
+## Installation
+
+Navigate to the Extensions page to install the extension and provide the
```

Comment:
Explain how to install. Provide a deeplink. Use Netlify-deployed install URL.

Reply:
Are you saying I should provide instructions on how to install a Contentful extension into the space? It seems like none of the other marketplace extensions have that in the README, I assumed because it would be taken care of by the marketplace UI itself.

Reply:
I'm adding more installation steps with a guide. Should be up soon.

Reply:
Done.